.github/workflows/macos_dependencies.txt

```diff
@@ -11,7 +11,6 @@ gettext
 ghostscript
 giflib
 git
-libjpeg-turbo
```

Comment:
Before I saw there was a duplicate of this dependant, I thought this was not necessary... Good catch!